neuraxle/base.py

```diff
@@ -180,7 +187,8 @@ def convolved_1d(self, stride, kernel_size) -> Iterable['DataContainer']:
         :return: an iterator of DataContainer
         :rtype: Iterable[DataContainer]
 
-        .. seealso:: `<https://github.com/guillaume-chevalier/python-conv-lib>`_
+        .. seealso::
+            `<https://github.com/guillaume-chevalier/python-conv-lib>`_
```

Comment:
Please read this: https://github.com/guillaume-chevalier/python-conv-lib/blob/master/testing/test_convolved_1d.py

Search for `include_incomplete_pass=True`, you should use this with true here to allow looping on batches smaller than the kernel size. E.g. if the kernel size is 7 but the batch is long of 3, normally it would have skipped everything! But with this to true, it'll include one smaller loop on the 3 items of partial batch.

Reply:
I know it's not in this pr, but this needs to be fixed anyway.

Reply:
Please open issue before resolving. this is important.

Reply:
I've set it to true, okey opening an issue, but I am not sure what you mean here will copy your comment in the issue

Reply:
created issue #161